doc/specs/Makefile.am

```diff
@@ -12,9 +12,9 @@ draft-morgan-pam-current.txt: padout draft-morgan-pam.raw
 AM_YFLAGS = -d
 
 CC = @CC_FOR_BUILD@
-CPPFLAGS = @BUILD_CPPFLAGS@
-CFLAGS = @BUILD_CFLAGS@
-LDFLAGS = @BUILD_LDFLAGS@
+AM_CPPFLAGS = @BUILD_CPPFLAGS@
+AM_CFLAGS = @BUILD_CFLAGS@
+AM_LDFLAGS = @BUILD_LDFLAGS@
```

Comment:
I'm not sure about this change.
Unlike all other parts of Linux-PAM, here a native compiler (@CC_FOR_BUILD@) is used, and it may have different options, e.g. @BUILD_CFLAGS@ might be incompatible with CFLAGS.
While GNU Automake complains about overriding CFLAGS, it doesn't complain about overriding CC, which is inconsistent and makes little sense.